subprojects/common/http-client/src/main/kotlin/com/avito/http/internal/StatsdServiceEventsListener.kt

```diff
@@ -0,0 +1,27 @@
+package com.avito.http.internal
+
+import com.avito.android.stats.CountMetric
+import com.avito.android.stats.SeriesName
+import com.avito.android.stats.StatsDSender
+import com.avito.android.stats.TimeMetric
+import java.io.IOException
+import java.net.SocketTimeoutException
+
+internal class StatsdServiceEventsListener(
+    private val statsDSender: StatsDSender,
+    private val prefix: SeriesName
+) : ServiceEventsListener {
+
+    override fun onResponse(code: Int, latencyMs: Long) {
+        statsDSender.send(CountMetric(prefix.append("response", "$code")))
+        statsDSender.send(TimeMetric(prefix.append("latency"), latencyMs))
+    }
```

Comment:
It looks that we don't need 2 metrics - response and latency. They are the same:

What is a response? It's a time metric with different results.

time: `<endpoint>.<result>`

- `api.method.200 : 25 ms`
- `api.method.502 : 10 ms`

Reply:
You're right, will do along with per-method metrics